molgenis-jobs/src/main/java/org/molgenis/data/jobs/JobExecutionContext.java

```diff
@@ -7,6 +7,11 @@
  */
 public class JobExecutionContext
 {
+	private JobExecutionContext()
+	{
+
+	}
+
```

Comment:
Sonatype says you should prevent instantiation of the JobExecutionContext by providing a private constructor. So doing that.